src/sql/util.py

```diff
@@ -251,17 +266,26 @@ def check_duplicate_arguments(
     else:
         duplicates_error = ""
 
-    if alias_pairs_present:
-        arg_list = sorted([" or ".join(pair) for pair in alias_pairs_present])
+    if alias_pairs_present and not len(delete_arg_present) > 1:
```

Comment:
The condition can be simplified for readability: `len(delete_arg_present) <= 1`

Reply:
@neelasha23 Fixed [here](https://github.com/ploomber/jupysql/pull/946/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R269)